dynamic_rest/processors.py

```diff
@@ -78,14 +78,18 @@ def process(self, obj, parent=None, parent_key=None, depth=0):
                     return
 
                 name = obj.serializer.get_plural_name()
-                pk = obj.instance.pk
+                pk = obj.pk_value or obj.instance.pk
```

Comment:
This changes the sideloading behavior to make sure the `pk` is replaced with an object that contains `type` and `id`. Basically DynamicGenericRelationField attaches the desired `pk` value to the `TaggedDict` object (`obj`), which is used here.